linux/installscripts/install4debian.sh

```diff
@@ -0,0 +1,78 @@
+
+#!/bin/sh
+# set DISTRO either to "Ubuntu", "Debian" or "Fedora"
+DISTRO="Debina"
```

Comment:
Debian/Debina

Reply:
@trebmuh  Thank you for fixing the typo. @corrados yes please move the scripts to the most appropriate place in the repo. Users will find the script mainly with a reference in the `README.md` or a reference in Wiki, because the majority will install and not contribute code.